packages/admin/cms-admin/src/common/routeMenu/routeMenu.type.ts

```diff
@@ -0,0 +1,11 @@
+import { MenuItemRouterLinkProps } from "@comet/admin";
+import { RouteProps } from "react-router";
+
+export type RouteMenuItem = Omit<MenuItemRouterLinkProps, "to"> & {
+    route?: RouteProps;
+    to?: string;
```

Comment:
Do I understand this correctly:
- either you specify a route, then a route is generated and used as link
- or you specify to, then only the link is used